sdk/core/azure-core/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ set(
     inc/azure/core/http/transport.hpp
     inc/azure/core/internal/contract.hpp
     inc/azure/core/internal/hkeyholder.hpp
+    inc/azure/core/internal/http/null_body_stream.hpp
```

Comment:
This won't get us much if we get rid of the internal namespace.

Reply:
@RickWinter let us know the concrete plan for `::Internals` with the rest of the namespace update.

Reply:
Are we removing Internal !!!???   That would be a shame!  🤣 
I believed the target was to remove `core` , so we would live with `azure/internal...`